qemu/tests/file_copy_between_blk_scsi.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from virttest import utils_misc
+from virttest import error_context
+from avocado.core import exceptions
```

Comment:
Instead of raising `exceptions.Test*()`, please call `test.fail() | test.error() | ...` directly.